resources/virt.go

```diff
@@ -49,18 +62,21 @@ type VirtRes struct {
 	Transient  bool               `yaml:"transient"` // defined (false) or undefined (true)
 	CPUs       uint16             `yaml:"cpus"`
 	Memory     uint64             `yaml:"memory"` // in KBytes
+	OSInit     string             `yaml:"osinit"` // init used by lxc
```

Comment:
Ah, right, good point. I had only used commands without additional parameters in my tests.

This will work as is. There are simple workarounds that can be done to avoid passing parameters so this will function as-is with room for future improvement.